Editor/UI/EditorWindows/EditorWindowBase.cs

```diff
@@ -12,35 +12,40 @@ namespace ReadyPlayerMe.Core.Editor
     public class EditorWindowBase : EditorWindow
     {
         private const string SUPPORT_HEADING = "Support";
-        private const string DOCS_URL = "https://bit.ly/UnitySDKDocs";
-        private const string FAQ_URL =
-            "https://docs.readyplayer.me/overview/frequently-asked-questions/game-engine-faq";
-        private const string DISCORD_URL = "https://bit.ly/UnitySDKDiscord";
+        private const string ERROR_ICON_SEARCH_FILTER = "t:Texture rpm_error_icon";
 
         protected GUIStyle HeadingStyle;
         protected GUIStyle DescriptionStyle;
+        
+        protected Texture errorIcon;
 
         private GUIStyle webButtonStyle;
 
         private readonly GUILayoutOption windowWidth = GUILayout.Width(460);
         protected readonly float ButtonHeight = 30f;
         private Banner banner;
+        private Footer footer;
 
         private string editorWindowName;
         private bool windowResized;
 
-#if !DISABLE_AUTO_INSTALLER
-        public static readonly string ERROR_IMAGE_PATH = "Packages/com.readyplayerme.core/Editor/error.png";
-#else
-        public static readonly string ERROR_IMAGE_PATH = "Assets/Ready Player Me/Core/Editor/error.png";
-#endif
-
-        /// <summary>
-        /// 
-        /// </summary>
         private void LoadAssets()
         {
-            banner ??= new Banner();
+            banner ??= new Banner(editorWindowName);
+            
+            footer ??= new Footer(editorWindowName);
+            
+            // TODO: move this into avatar loader settings window
```

Comment:
para-phrasing a quote from Pablo "TODO's never get done" :D probably we could remove

Reply:
shit forgot this here, will take a look now